CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.5.1] - Unreleased
```

Comment:
Given that you are changing behavior here (the log type and the message format), versus just a small bug fix.. I think this deserves `2.6.0` - sorry I missed this before

Reply:
Ah I had thought this was a bug 😅 Sounds good to me.